backend/apis/github-issues.api.js

```diff
@@ -37,9 +37,7 @@ app.post("/api/github-issues", (req, res, next) => {
     .then(response => {
       res.status(200).send({
         issueNumber: response.data.number,
-        issueUrl: `https://github.com/JustUtahCoders/comunidades-unidas-internal/issues/${
-          response.data.number
-        }`
+        issueUrl: `https://github.com/JustUtahCoders/comunidades-unidas-internal/issues/${response.data.number}`
```

Comment:
Change here is formatting only and is related to upgrading prettier to latest version